### PR TITLE
Lps 153120

### PR DESCRIPTION
**Pull request by SamZiemer**
https://issues.liferay.com/browse/LPS-153120

/cc @thienquach87

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by liuxingye** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
@SamZiemer Can you take a look at the test failures? Thanks!

File: modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/display/context/DLEditFileEntryDisplayContext.java [hunk 1]
```diff
@@ -26,6 +27,9 @@
 public interface DLEditFileEntryDisplayContext extends DLDisplayContext {
 
 	public DDMFormValues getDDMFormValues(long classPK) throws PortalException;
+	public default String getAllMimeTypeSizeLimit() throws PortalException {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/display/context/DLEditFileEntryDisplayContext.java:30]: SF needed here

**Comment by ambrinchaudhary** [comment]
Closing in favor of https://github.com/liferay-lima/liferay-portal/pull/3282

**Pull request closed without merging**